### ISSUE TITLE
Text color of bonus levels is hard to distinguish

### ISSUE THREAD

**Issue by Wuzzy2** [Neverball/neverball#116]
In Neverball 1.6.0, the colors for available bonus levels are hard to distinguish:

* Solved levels are green
* Unsolved but available levels are also green, but with a very slight gradient to yellow

Those two colors look almost identical, I propose to tweak the colors to make them easier to distinguish.

**Comment by Wuzzy2** [comment]
Note: Implementing #123 will automatically fix this issue.

**Comment by KodeBreaker-Has-SWAG** [comment]
I agree, I think unbeaten bonus levels should be like a blue/purple gradient color
